src/DynamoPackages/PackageLoader.cs

```diff
@@ -158,8 +158,10 @@ private IEnumerable<CustomNodeInfo> OnRequestLoadCustomNodeDirectory(string dire
 
         /// <summary>
         ///     Load the package into Dynamo (including all node libraries and custom nodes)
-        ///     and add to LocalPackages
+        ///     and add to LocalPackages.
+        ///     NOTE: This method DOES NOT import ZT packages into Dynamo VM. Use LoadPackages() instead.
```

Comment:
is this a change in behavior after your PR?

Reply:
Yes, this is a change in behavior of the API. I will try to obsolete this and create a new method for internal use.